bot/exts/info/patreon.py

```diff
@@ -0,0 +1,160 @@
+
+        embed_list = []
+
+        embed_patron_info = discord.Embed(
+            title="Patreon Supporters",
+            description=(
+                "Here is a full list of this months Python Discord patrons!\n\nWe use the money from Patreon to offer "
+                "excellent prizes for all of our events. Prizes like t-shirts, stickers, microcontrollers that support "
+                "CircuitPython, or maybe even a mechanical keyboard.\n\nYou can read more about how Patreon supports "
+                "us, or even support us yourself, on our Patreon page [here](https://pydis.com/patreon)!"
+            )
+        )
+
+        embed_list.append(embed_patron_info)
+
+        if tier_1_patrons:
+            embed_tier_1 = discord.Embed(
+                title="Tier 1 patrons",
+                description="\n".join(tier_1_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_1).colour
+            )
+            embed_list.append(embed_tier_1)
+
+        if tier_2_patrons:
+            embed_tier_2 = discord.Embed(
+                title="Tier 2 patrons",
+                description="\n".join(tier_2_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_2).colour
+            )
+            embed_list.append(embed_tier_2)
+
+        if tier_3_patrons:
+            embed_tier_3 = discord.Embed(
+                title="Tier 3 patrons",
+                description="\n".join(tier_3_patrons),
+                colour=guild.get_role(constants.Roles.patreon_tier_3).colour
+            )
+            embed_list.append(embed_tier_3)
+
+        await channel.send(embeds=embed_list)
+
+    @commands.command("patrons")
+    async def current_supporters_command(self, ctx: commands.Context) -> None:
+        """Sends the current list of patreon supporters, sorted by tier level."""
+        await self.send_current_supporters(ctx.channel)
+
+    @tasks.loop(time=datetime.time(hour=17))
+    async def current_monthly_supporters(self) -> None:
+        """A loop running daily to see if it's the first of the month. If so call `self.send_current_supporters()`."""
+        date = datetime.date.today().day
+
+        if date == 1:
+            await self.send_current_supporters(
+                utils.channels.get_or_fetch_channel(constants.Channels.meta)
+            )
+
+    @commands.command("patreon")
+    async def patreon_info(self, ctx: commands.Context) -> None:
+        """Sends patreon info."""
+        await ctx.send(embed=discord.Embed(
+            title="Patreon",
+            description=(
+                "We use the money from Patreon to offer excellent prizes for all of our events. Prizes like t-shirts, "
+                "stickers, microcontrollers that support CircuitPython, or maybe even a mechanical keyboard.\n\nYou can"
+                " read more about how Patreon supports us, or even support us yourself, on our Patreon page "
```

Comment:
```suggestion
                "stickers, microcontrollers that support CircuitPython, or maybe even a mechanical keyboard.\n\n"

                "You can" read more about how Patreon supports us, or even support us yourself, on our Patreon page "
```

I like to refect empty lines in the code.